src/devices/bus/nubus/nubus_48gc.cpp

```diff
@@ -79,7 +79,7 @@ jmfb_device::jmfb_device(const machine_config &mconfig, device_type type, const
 	m_is824(is824),
 	m_assembled_tag(util::string_format("%s:%s", tag, GC48_SCREEN_NAME))
 {
-	static_set_screen(*this, m_assembled_tag.c_str());
+	this->set_screen(m_assembled_tag.c_str());
```

Comment:
Do you need the explicit `this->` for cases like this?

Reply:
No, I don't. I will fix those